src/webgl/context-state.js

```diff
@@ -106,7 +106,7 @@ const GL_STATE = {
 
   depthRange: {
     type: new Float32Array(2),
-    value: new Float32Array(null, null), // TBD
+    value: new Float32Array(2), // TBD
```

Comment:
It is supposed to be the default value for `depthRange` for a new context, so that we don't have to query it. 

I think it should be `new FloatArray([0, 1])`